test/distributed/org/apache/cassandra/distributed/test/SchemaTest.java

```diff
@@ -197,4 +203,114 @@ private static boolean checkTablesPropagated(IInvokableInstance instance, boolea
                    && (Schema.instance.getTableMetadata(KEYSPACE, TABLE_TWO) != null ^ !two);
         });
     }
+
+    @Test
+    public void dropColumnMixedMode() throws Throwable
+    {
+        try (Cluster cluster = init(Cluster.build(2).start()))
+        {
+            cluster.schemaChange(withKeyspace("CREATE TABLE %s.tbl (id int primary key, v1 int, v2 int, v3 int)"));
```

Comment:
Inserts could easily be a loop

Reply:
Those tests were using loops and I removed them because it was more confusing than anything specially for the other test. I can simplify it by using a loop for inserting each row.

Reply:
Yep inserts is what I referring to, otherwise I read all inserts expecting some minor difference somewhere I am missing only to realize they're all the same.